Explorer/Assets/DCL_RenderPass_DrawSkyBox.cs

```diff
@@ -0,0 +1,227 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public partial class DCL_RenderFeature_ProceduralSkyBox : ScriptableRendererFeature
+{
+    public class DCL_RenderPass_DrawSkyBox : ScriptableRenderPass
+    {
+        //public static RTHandle k_CameraTarget
+        const string profilerTag = "Custom Pass: DrawSkyBox";
+        private Material m_Material_Draw;
+        private ProceduralSkyBoxSettings_Draw m_Settings_Draw;
+        RTHandle m_SkyBoxCubeMap_RTHandle;
+        RTHandle m_cameraColorTarget_RTHandle;
+        RTHandle m_cameraDepthTarget_RTHandle;
+        //private ScriptableRenderer m_Renderer = null;
+        Mesh m_cubeMesh = null;
+
+        private const string k_SkyBoxCubemapTextureName = "_SkyBox_Cubemap_Texture";
+        private static readonly int s_SkyBoxCubemapTextureID = Shader.PropertyToID(k_SkyBoxCubemapTextureName);
+
+        private enum ShaderPasses
```

Comment:
What is this enum for?

Reply:
Yeah it's for the shader passes, wasn't in use, but I've made changes so now it is